syntax = "proto3";

package orders;

service OrderService {
  rpc CreateOrder (OrderRequest) returns (OrderReply) {}
  rpc GetOrderStatus (OrderStatusRequest) returns (OrderStatusReply) {}
}

message OrderRequest {
  string order_id = 1;
  string description = 2;
  double value = 3;
}

message OrderReply {
  string message = 1;
  string order_id = 2;
}

message OrderStatusRequest {
  string order_id = 1;
}

message OrderStatusReply {
  string order_id = 1;
  string status = 2;
}